src/index.html

```diff
@@ -1,14 +1,83 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700;900&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <title>Moyo header</title>
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1 class="moyo-header">Moyo header</h1>
+    <header class="header" id="header">
+      <div class="header__list">
+        <div class="list__logo">
+          <a href="#header" class="logo__link">
+            <img src="./images/logo.png" alt="logo">
+          </a>
+        </div>
+
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item ">
+              <a
+              href="#apple"
+              class="nav__link is-active"
+              >Apple</a>
+            </li>
+            <li class="nav__item">
+              <a
+                href="#samsung"
+                class="nav__link"
+                >Samsung</a>
```

Comment:
```suggestion
              <a
                href="#samsung"
                class="nav__link"
              >
                Samsung
              </a>
```
closing bracket should be on the same level as opening one